server/server.js

```diff
@@ -226,31 +321,35 @@ app.listen(PORT, () => {
 });
 
-// Load environment variables from .env file.
-// Import necessary libraries (express, cors, bcryptjs, jsonwebtoken, request, and @prisma/client).
-// Express App Initialization:
 
-// Initialize the Express application and Prisma Client.
-// Enable CORS for all routes to allow cross-origin requests.
-// Use middleware to parse JSON request bodies.
-// Spotify API Configuration:
 
-// Set up constants for Spotify API URLs and credentials.
-// User Signup Endpoint (/signup):
 
-// Handles user registration by checking for existing users, hashing the password, and creating a new user in the database.
-// User Login Endpoint (/login):
 
-// Handles user login by verifying email and password, generating a JWT token, and managing user sessions in the database.
-// Protected Route Example (/protected):
+  // Load environment variables from .env file.
+  // Import necessary libraries (express, cors, bcryptjs, jsonwebtoken, request, and @prisma/client).
+  // Express App Initialization:
+
+  // Initialize the Express application and Prisma Client.
+  // Enable CORS for all routes to allow cross-origin requests.
+  // Use middleware to parse JSON request bodies.
+  // Spotify API Configuration:
+
+  // Set up constants for Spotify API URLs and credentials.
+  // User Signup Endpoint (/signup):
+
+  // Handles user registration by checking for existing users, hashing the password, and creating a new user in the database.
+  // User Login Endpoint (/login):
+
+  // Handles user login by verifying email and password, generating a JWT token, and managing user sessions in the database.
+  // Protected Route Example (/protected):
 
-// A sample protected route that requires a valid JWT token to access and responds with the user ID.
-// Spotify Authentication Endpoints:
+  // A sample protected route that requires a valid JWT token to access and responds with the user ID.
+  // Spotify Authentication Endpoints:
 
-// /auth/login: Redirects users to Spotify for authorization.
-// /auth/callback: Handles the callback from Spotify, exchanges the authorization code for access and refresh tokens, and redirects to the frontend.
-// /auth/token: Returns the access and refresh tokens if present.
-// /refresh_token: Refreshes the access token using the refresh token.
-// Server Initialization:
+  // /auth/login: Redirects users to Spotify for authorization.
+  // /auth/callback: Handles the callback from Spotify, exchanges the authorization code for access and refresh tokens, and redirects to the frontend.
+  // /auth/token: Returns the access and refresh tokens if present.
+  // /refresh_token: Refreshes the access token using the refresh token.
+  // Server Initialization:
 
-// Sets the server to listen on a specified port or default to port 3000 and logs that the server is running.
+  // Sets the server to listen on a specified port or default to port 3000 and logs that the server is running.
```

Comment:
Are those comments from Spotify?  Pls delete/update/add accordingly.

Reply:
this where from me to understand, I wrote it on my code, but it started getting too crowded so I moved it down for my code to actually be more readable, should i remove it?